MobileTraining/MobileTraining/Views/Login/LoginView.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  LoginView.swift
+//  MobileTraining
+//
+//  Created by Svetoslav Donkov on 14.05.23.
+//
+
+import SwiftUI
+
+struct LoginView: View {
+    @Binding var isLoggedIn: Bool
+    @ObservedObject var loginViewModel: LoginViewModel
+    @State private var showingAlert = false
+    @State private var isSecure = true
+    
+    private let networkManager: NetworkManager
```

Comment:
The network manager has no place in the view, nor the view model. It should be accessed only by the repositories via dependency injection.